ambari/user-guide/manage-hosts.rst

```diff
@@ -0,0 +1,230 @@
+Managing Hosts
+==============
+
+Use Ambari Hosts to manage multiple |rbp-stack| components such as DataNodes, NameNodes, CassandraNodes and AlluxioNodes, running on hosts throughout your cluster. For example, you can restart all DataNode components, optionally controlling that task with rolling restarts. Ambari Hosts supports filtering your selection of host components, based on operating status, host health, and defined host groupings.
```

Comment:
Since the topic is quite large, adding a index may help user to preview arguments at glance.